tests/operator_test.go

```diff
@@ -674,7 +672,7 @@ var _ = Describe("ALL Operator tests", func() {
 		var _ = Describe("Alert tests", func() {
 			f := framework.NewFramework("alert-tests")
 
-			It("CdiOperatorDown alert firing when operator scaled down", func() {
+			It("CDIOperatorDown alert firing when operator scaled down", func() {
 				if !utils.IsOpenshift(f.K8sClient) {
```

Comment:
Same not sure if this openshift specific at this point. kubevirtci has a prometheus option now.